mockgen/mockgen.go

```diff
@@ -289,6 +289,8 @@ func (g *generator) Generate(pkg *model.Package, outputPkgName string, outputPac
 		g.p("// Source: %v (interfaces: %v)", g.srcPackage, g.srcInterfaces)
 	}
 	g.p("")
+	g.p("// Generated by this command:")
+	g.p("//    %v", strings.Join(os.Args, " "))
```

Comment:
Let's make this part of the same comment block. Otherwise the new comment block becomes separated from the rest of the package level comment.

```suggestion
	g.p("//")
	g.p("// Generated by this command:")
	g.p("//    %v", strings.Join(os.Args, " "))
```